metrics_azurerm_resources.go

```diff
@@ -105,21 +111,69 @@ func (m *MetricsCollectorAzureRmResources) collectAzureResources(subscription *a
 
 	resourceMetric := m.Collector.GetMetricList("resource")
 
+	// Get azure disk detail
```

Comment:
i'm currently not sure if we should add disks to the resourcemanager exporter.

This might be a better solution to use the azure-resourcegraph-exporter when you can use a ResourceGraph Kusto query to get every list you want.

Otherwise i would have to add every resource to this exporter and that's the reason most of the stuff was removed from this exporter. the resourcegraph queries are more flexible than this exporter.

Reply:
ok for this part i understand, ca we ignore this modifications on this file ?